src/eei.cpp

```diff
@@ -347,6 +382,8 @@ namespace HeraVM {
 
       context->fn_table->log(context, &msg.address, data.data(), length, topics, numberOfTopics);
 
+      takeGas(GasSchedule::log + (length * GasSchedule::logData) + (GasSchedule::logTopic * numberOfTopics));
```

Comment:
How do you prevent overflows?

Reply:
I don't see a solution to this that isn't messy.
Maybe a helper function would be better for this.